test/regression/cases/file_to_blackhole_1000ms_latency/experiment.yaml

```diff
@@ -31,3 +31,9 @@ checks:
       series: total_rss_bytes
       # The machine has 12GiB free.
       upper_bound: 1.2GiB
+
+  - name: lost_bytes
+    description: "Allowable bytes not polled by log Agent"
+    bounds:
+      series: lost_bytes
+      upper_bound: 0KB
```

Comment:
Minor nit: would suggest using `0KiB` for consistency with the existing checks. The experiment configs look good to me. Thanks for adding these experiment configs!

Reply:
Fixed!